openupgrade_scripts/scripts/calendar/14.0.1.0/post-migration.py

```diff
@@ -40,7 +40,10 @@ def fill_calendar_recurrence_table(env):
             FROM calendar_event
             WHERE recurrency AND recurrence_id IS NULL
                 AND (recurrent_id IS NULL OR recurrent_id = 0)
-                AND (mo OR tu OR we OR th OR fr OR sa OR su)
+                AND (
+                        rrule_type != 'weekly'
+                        OR (mo OR tu OR we OR th OR fr OR sa OR su)
+                    )
```

Comment:
```suggestion
                AND (
                    rrule_type != 'weekly'
                    OR (
                        rrule_type = 'weekly' AND
                        (mo OR tu OR we OR th OR fr OR sa OR su)
                    )
                )
```

Reply:
if `rrule_type != 'weekly'` then `rrule_type = 'weekly'`, why you need to check `rrule_type = 'weekly' AND (mo OR tu OR we OR th OR fr OR sa OR su)`

Reply:
Because you may have marked the event as weekly recurrent, but then no recurrent day is selected. That's possible on <=v13, but on v14, there's a constraint avoiding that, so we have to avoid to insert such fake recurrent events.

Reply:
ok. I fixed it